File: Landowner/src/landowner/ui/console/ExceptionOperation.java
package landowner.ui.console;

public class ExceptionOperation implements OperationRole {

	public void run() {
		System.out.println("Chose a valid option.");
	}

	public String getMessage() {
		return "";
	}

	public boolean doContinue() {
		return true;
	}

}

File: caphyon/source/ExceptionOperation.cpp
#include"../includes/ExceptionOperation.h"
#include<iostream>
using namespace std;

void ExceptionOperation::run() {
	cout << "\nChoose a valid option.\n";
}

string ExceptionOperation::getMessage() {
	return "";
}

bool ExceptionOperation::doContinue() {
	return true;
}